src/components/FormInput.jsx

```diff
@@ -1,7 +1,7 @@
 import styled from '@emotion/styled';
 
 export const FormInput = styled.input`
-  width: 268px;
+  max-width: 100%;
```

Comment:
You can left width 268px, max-width is only to avoid that small screen has overflow content
```suggestion
  width: 268px;
  max-width: 100%;
```